disnake/member.py

```diff
@@ -756,7 +756,9 @@ async def ban(
         if delete_message_days is not MISSING:
             await self.guild.ban(self, reason=reason, delete_message_days=delete_message_days)
         else:
-            await self.guild.ban(self, reason=reason, delete_message_seconds=delete_message_seconds)
+            await self.guild.ban(
+                self, reason=reason, delete_message_duration=delete_message_duration
+            )
```

Comment:
as far as I can tell, this is not necessary. Because delete_message_days defaults to MISSING, we can pass them all directly through to guild.ban.

Reply:
At runtime, yup, but pyright would otherwise complain about there not being a matching overload if both parameters are provided in the same call :/

Reply:
just ignore it